formapp/app.py

```diff
@@ -2169,7 +2173,8 @@ def endpoint_home():
     if loginResult:
         return loginResult
     else:
-        return render_template("index.html")
+        className = sD.studentDict[request.remote_addr]['class']
+        return render_template("index.html", className = className)
```

Comment:
Please test that this will work without a value in sD.studentDict[request.remote_addr]['class'], or that it is not possible to sign in without a class.

It should say "Formbar" by default.

Reply:
It is not possible to sign in without a class anymore

Reply:
Ethan is doing that now